RubeesEat/Pages/Shared/_Layout.cshtml

```diff
@@ -22,7 +22,7 @@
             <a href="/SplitBill">Split a bill</a>
         </li>
         <li>
-            <a href="/LookBill">Show bills</a>

```

Comment:
`/ShowBills/1` looks like we try to look at a bill with the ID 1 but it indicates the page instead. I think we should do two things here:

1. Use the query to provide the page (e.g. `/ShowBills?page=1`) instead of the route.
2. Add a default to the value so that we don't have to provide it here. This way, we can use `/ShowBills` as link and it will automatically use the first page